### DISCUSSION TITLE
QRZ/HamQTH Lookups not working

### DISCUSSION THREAD

**Discussion by minorsecond** [magicbug/Cloudlog#1293] [General]
I'm on the latest version as of today and when I enter a call in the call field, the info is not pulled either from hamQTH or QRZ. I've tried both in the config file. I'm a QRZ XML subscriber and the API key is set in the GUI. I've verified that CURL is installed as well. Is there anything else I can check?

**Comment by minorsecond** [comment]
I just realized I missed the entries in the config.php file for qrz username and qrz password.

**Comment by magicbug** [comment]
Pleased you figured it out :)
